src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -0,0 +1,175 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.service
```

Comment:
I assume this is just a rename

Reply:
split the visit, session, session-template & support.  session and session-template are really the same thing but split it based on the endpoints.